Source/Core/DolphinQt/Config/Mapping/WiimoteEmuGeneral.cpp

```diff
@@ -66,10 +67,11 @@ void WiimoteEmuGeneral::CreateMainLayout()
   setLayout(m_main_layout);
 }
 
-void WiimoteEmuGeneral::Connect()
+void WiimoteEmuGeneral::Connect(MappingWindow* window)
 {
   connect(m_extension_combo, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged),
           this, &WiimoteEmuGeneral::OnAttachmentChanged);
+  connect(window, &MappingWindow::Update, this, &WiimoteEmuGeneral::Update);
```

Comment:
Just like how in the base MappingWidget class, when MappingWindow::Update is called, MappingWidget::Update gets called to update its widgets, we also want to update WiimoteEmuGeneral's widgets.